components/raftstore-v2/src/operation/command/admin/split.rs

```diff
@@ -343,16 +349,19 @@ impl<EK: KvEngine, R: ApplyResReporter> Apply<EK, R> {
         }
 
         let derived_path = self.tablet_registry().tablet_path(region_id, log_index);
-        checkpointer
-            .create_at(&derived_path, None, 0)
-            .unwrap_or_else(|e| {
-                panic!(
-                    "{:?} fails to create checkpoint with path {:?}: {:?}",
-                    self.logger.list(),
-                    derived_path,
-                    e
-                )
-            });
+        // If it's recovered from restart, it's possible the target path exists already.
```

Comment:
Add a comment somewhere to explain checkpoint is atomic, if there isn't one already.